Makefile.devel

```diff
@@ -89,14 +89,14 @@ DEVELOPERS=\
 # PRERELEASE_PREFIX is used to match PRERELEASE_TAG in sed
 # line in tag target below, which edits $(DOWNLOADHTML)
 # The prereltag.txt is kept as a record in the distrib area
-# of the current pre-release version (currently not used explicitly 
+# of the current pre-release version (currently not used explicitly
 # anywhere for web pages/whatever).
-PRERELEASE_PREFIX=4\.4\.1~pre
-PRERELEASE_TAG=4.4.1~pre
+RERELEASE_PREFIX=4\.4\.1-pre
```

Comment:
Hi @zhenya1007,
Regarding these two lines, in addition to my suggestion in https://github.com/ProofGeneral/PG/pull/127#discussion_r94273448, there is a missing P in "RERELEASE_PREFIX" ;-)